mbbsd/mail.c

```diff
@@ -1227,6 +1253,8 @@ mail_account_sysop(void)
     char            save_title[STRLEN];
     char            tmp_title[STRLEN-20];
```

Comment:
This implementation is copied&pasted from the function "do_send", but I don't know why STRLEN-20 and what is 20.